src/browser/declarativewebutils.cpp

```diff
@@ -75,13 +77,14 @@ void DeclarativeWebUtils::clearStartupCacheIfNeeded()
 {
     QFileInfo systemStamp(gSystemComponentsTimeStamp);
     if (systemStamp.exists()) {
-        QString mostProfilePath = QDir::homePath() + gProfilePath;
-        QString localStampString(mostProfilePath + QString("/_CACHE_CLEAN_"));
-        QFileInfo localStamp(localStampString);
-        if (localStamp.exists() && systemStamp.lastModified() > localStamp.lastModified()) {
-            QDir cacheDir(mostProfilePath + "/startupCache");
-            cacheDir.removeRecursively();
-            QFile(localStampString).remove();
+        QString startupCacheDir = QDir::homePath() + gStartupCacheDir;
+        QDir cacheDir(startupCacheDir);
```

Comment:
I think it's better to omit intermediate `QString startupCacheDir` variable. Then init `QDir cacheDir(QDir::homePath() + gStartupCacheDir);`.

Reply:
Thanks, yes, the intermediate string is unnecessary. I've combined the two lines.